portfolio/src/main/webapp/script.js

```diff
@@ -63,7 +63,15 @@ function displayCaption(img) {
  */
 async function getMessageFromServlet(){
   const response = await fetch('/data');
-  const hello = await response.text();
-  document.getElementById('message-container').innerText = hello;
+  const messageData = await response.json();
+  console.log(messageData);
+  const messages = document.getElementById('message-container');
+  messages.innerHTML = '';
+  messageData.forEach(message => messages.appendChild(createParagraphElement(message)));
+}
 
+function createParagraphElement(text){
+  const pElement = document.createElement('p');
```

Comment:
rename this to paragraphElement. In general, try to avoid abbreviations in naming variables, unless it's a well-known abbreviation.

Reply:
Got it!

Reply:
Whoops, just kidding. Second commit is coming in a second.